manim/animation/creation.py

```diff
@@ -397,6 +398,62 @@ def __init__(
         )
 
 
+class SpiralIn(Animation):
+    r"""Create the Mobject with sub-Mobjects flying in on spiral trajectories.
+
+    Parameters
+    ----------
+    shapes
+        The MObject on which to be operated.
+
+    scale_factor
+        The factor used for scaling the effect.
+
+    Examples
+    --------
+    .. manim :: SpiralInExample
+
+        class SpiralInExample(Scene):
+            def construct(self):
+                pi = MathTex(r"\pi").scale(7)
+                pi.shift(2.25 * LEFT + 1.5 * UP)
+                circle = Circle(color=GREEN_C, fill_opacity=1).shift(LEFT)
+                square = Square(color=BLUE_D, fill_opacity=1).shift(UP)
+                shapes = VGroup(pi, circle, square)
+                self.play(SpiralIn(shapes))
+    """
+
+    def __init__(
+        self,
+        shapes: Mobject,
+        scale_factor: float = 1,
+        **kwargs,
+    ) -> None:
+        self.shapes = shapes
+        self.scale_factor = scale_factor
+        expansion_factor = 8 * self.scale_factor
+        self.shape_center = shapes.get_center()
+        for shape in shapes:
+            shape.final_position = shape.get_center()
+            shape.initial_position = (
+                shape.final_position
+                + (shape.final_position - self.shape_center) * expansion_factor
+            )
+            shape.move_to(shape.initial_position)
+            shape.save_state()
+
+        super().__init__(shapes, **kwargs)
+
+    def interpolate_mobject(self, alpha: float) -> None:
+        TAU = 2 * np.pi
```

Comment:
`TAU` is already defined in `constants.py`, no need to redefine it here.

Reply:
I 100% agree. I shouldn't be redefining TAU here. I better fix this; thanks for the suggestion.

Reply:
Hey @Darylgolden, constants.py isn't imported here, and I'm having trouble importing constants.py due to its location relative to manim/animation/creation.py. Any thoughts?

Reply:
Okay, this is done. 👍🏻